BaseApiBundle/DependencyInjection/Compiler/SerializerCompilerPass.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace OpenOrchestra\BaseApiBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Class SerializerCompilerPass
+ */
+class SerializerCompilerPass implements CompilerPassInterface
```

Comment:
Why using a compiler pass, it seems that you can set parameters directly in OpenOrchestraBaseApiExtension.php